spec/javascripts/components/story/story_item_spec.js

```diff
@@ -80,4 +80,34 @@ describe('<StoryItem />', () => {
       expect(children).toHaveProp('title', title);
     });
   });
+
+  describe('when the story is accepted', () => {
+    it('put .Story--accepted on childrens prop className', () => {
+      const story = storyFactory({
+        collapsed: false,
+        state: 'accepted'
+      });
+      const className = 'Story--accepted';
+
+      const wrapper = shallow(<StoryItem story={story} />);
+      const children = wrapper.find(ExpandedStory);
+
+      expect(children).toHaveProp('className', className);
+    });
+  });
+
+  describe('when the story is not accepted', () => {
+    it('do not put .Story--accepted on childrens prop className', () => {
+      const story = storyFactory({
+        collapsed: false,
+        state: 'unscheduled'
+      });
+      const className = 'Story--accepted';
+
+      const wrapper = shallow(<StoryItem story={story} />);
+      const children = wrapper.find(ExpandedStory);
```

Comment:
You can move it too.
Another thing, I think you don't need the variable `className` here just to use it in the `expect`.
```js
const className = 'Story--accepted';
...
expect(children).not.toHaveProp('className', className);
// I believe this way is simple :D
expect(children).not.toHaveProp('className', 'Story--accepted');
```